webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala

```diff
@@ -146,7 +146,9 @@ class PrepareTemplateVariablesImpl(
         .toVariable(if (agentNodeConfig.agentType == AgentType.CfeCommunity) Seq("true") else Seq()),
       systemVariableSpecService.get("AGENT_TYPE").toVariable(Seq(agentNodeConfig.agentType.toString)),
       systemVariableSpecService.get("RUDDER_NODE_CONFIG_ID").toVariable(Seq(nodeConfigVersion.value)),
-      systemVariableSpecService.get("RUDDER_COMPLIANCE_MODE").toVariable(Seq(agentPolicyMode.name))
+      systemVariableSpecService.get("RUDDER_COMPLIANCE_MODE").toVariable(Seq(agentPolicyMode.name)),
+      // For compatibility with pre-8.0 agents
+      systemVariableSpecService.get("RUDDER_VERIFY_CERTIFICATES").toVariable(Seq("false"))
```

Comment:
You need to revert what was in `SystemVariableSpecImpl` (https://github.com/Normation/rudder/pull/4982/files#diff-b2bb87fe54627ab29d48267c7a88d069948c01442c48aeeaeaa1104cb1e441a6) to have that working

Reply:
Perhaps you can create the spec it here in an ad-hoc way with a comment "for compat, will be removed in 8.1" (or when it's time)

Reply:
The tests seem to pass without it, is it expected?

Reply:
No ? (I'm not even sure why you didn't get a "missing system variable spec" error)